.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Firebase Data Connect
+    url: https://github.com/firebase/firebase-ios-sdk/issues/new/choose
```

Comment:
Just a reminder that we should add Data Connect to this list https://github.com/firebase/firebase-ios-sdk/blob/a70e2e3cba4381559d98fcc672e0ba0cafd57885/.github/ISSUE_TEMPLATE/BUG_REPORT.yml?plain=1#L64-L89.

Reply:
Good catch! https://github.com/firebase/firebase-ios-sdk/pull/13657